src/controller/python/matter/clusters/Objects.py

```
@@ -35651,6 +35655,38 @@ def attribute_type(cls) -> ClusterObjectFieldDescriptor:
 
             value: typing.Union[None, Nullable, uint] = None
 
+        @dataclass
+        class CriticalFreezeProtection(ClusterAttributeDescriptor):
+            @ChipUtility.classproperty
+            def cluster_id(cls) -> int:
```

Pylint (4.0.7)

[error] Method 'cluster_id' should have "self" as first argument

(E0213)